.github/workflows/update-codeclimate-coverage.yml

```diff
@@ -0,0 +1,27 @@
+# Summary:
+# Run Unit and coverage tests, then upload it to Code Climate dashboard
+# Read ./README.md for extensive documentation
+
+name: Update Code Climate Coverage
+
+on:
+  push:
+    branches:
+      - 'v2-mst-aptd-gcms-lcz-sty'
+
+jobs:
+  run-tests-coverage:
```

Comment:
Why not keeping the same names of tests as the others?

https://github.com/UnlyEd/next-right-now/blob/df72481c54d28bbddf0ef3431a76e77899b53272/.github/workflows/deploy-zeit-staging.yml#L15-L22

This would be better if it uses the same "structure". First, setup env, then perform actions.

Reply:
Yes and no... Create many jobs make the workflow longer.
In staging, we are doing it first, so every other jobs can use it.
I don't think it's bad but it's up to you!

Reply:
It doesn't really matter if it's 5s longer (we aren't waiting for this task to perform another), readability matters more. using the same "setup" in all actions feels better.

Reply:
You're the boss! And the one who has to work with :p

Reply:
![image](https://user-images.githubusercontent.com/24303525/84035698-0efdf600-a994-11ea-8790-00d88c407622.png)
@Vadorequest install dependencies doesn't work when it's done in another job. We will still setup first node.js but we will install dependencies in this job...

Reply:
It's not an issue, and it's even better IMHO. This way, we know how stuff relates together. It forces to make it "clean" and centralise together those kinds of things.

Reply:
Yes and that's why we need to checkout the code in each job @Vadorequest  :thinking: